test/integration/perf_nsmd_deploy_test.go

```diff
@@ -0,0 +1,36 @@
+// +build basic
```

Comment:
please name the file according to the tag and vice versa.

Reply:
Moved to basic tests actually.